### DISCUSSION TITLE
解决两个 Android 模拟器之间无法网络通信的问题 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#208] [Announcements]
https://mazhuang.org/2017/12/03/tcp-connect-between-android-emulators/ 

 本文解决的是一个小众场景的问题：

**Comment by DaveGeneral** [comment]
测试测试，看看评论存储在哪里

**Comment by importpy** [comment]
测试

**Comment by GarfieldJiang** [comment]
楼上是在调戏po主么……

**Comment by mzlogin** [comment]
@GarfieldJiang 是的 😢

**Comment by importpy** [comment]
博主快来看，我的评论区有问题，点击评论后直接跳转博客，登陆不成功，怎么回事啊
[`地址`](http:importpy.github.io)

**Comment by lvxiaoxin** [comment]
炫酷的评论功能

**Comment by mzlogin** [comment]
@importpy 你这应该是申请 GitHub Application 那一步填的 Authorization callback URL 与你的域名不匹配。

**Comment by importpy** [comment]
我填写的就是http：//importpy.github.io  这个不对么？？

**Comment by mzlogin** [comment]
@importpy `https://importpy.github.io`，加个 s 试试

**Comment by importpy** [comment]
@mzlogin  感谢，成功解决

**Comment by mzlogin** [comment]
@importpy nice~不客气。

**Comment by qiuyimo** [comment]
nice

**Comment by bugren** [comment]
测试

**Comment by Johnson0608** [comment]
测试

**Comment by 18811707971** [comment]
大佬好，我前面fork您的工程，也搭建了一个博客，这几天发现图片无法显示了，请教一下您是哪个地方出了问题吗？

**Comment by mzlogin** [comment]
> @18811707971
> 大佬好，我前面fork您的工程，也搭建了一个博客，这几天发现图片无法显示了，请教一下您是哪个地瓜出了问题吗？

我看你博客里引用的主要是 CSDN 的图片链接，在 GitHub 仓库页面能正常显示，在你的博客页面无法正常显示，我猜可能是 CSDN 的防盗链机制限制了在你域名的显示。

你可以试试把图片放到 GitHub 仓库里，然后引用本地链接。

**Comment by 18811707971** [comment]
好的，谢谢前辈，晚上回去测试一下。

**Comment by zhangtuodd** [comment]
l试试
